### DISCUSSION TITLE
Error

### DISCUSSION THREAD

**Discussion by sharaf81995** [firemodels/fds#13432] [General]
hay all ,

I'm encountering an error when I create the wind profile in a .csv file and run my simulation in FDS 6.9.0.
 The error message is:

At line 5066 of file ../Source/init.f90
Fortran runtime error: End of file

Error termination. Backtrace:
At line 5066 of file ../Source/init.f90
At line 5066 of file ../Source/init.f90
Fortran runtime error: End of file

Has anyone experienced this or can guide me on how to resolve it?

Regards,
Sharaf

**Comment by drjfloyd** [comment]
We need more information. Please, simplify your fds input file as much as possible while still getting the error and attach it and the wind csv file.

**Comment by Er9y714** [comment]
Might it be due to csv file? See this:
https://github.com/firemodels/fds/issues/12780#issuecomment-2051779235
